backend/tests/data_layer/database/crud/postgresql/test_user_crud.py

```
@@ -0,0 +1,200 @@
+    
+
+
+def test_is_attr_data_in_db(session):
+    """
+    Test if attribute data exists in the database.
+    Verifies that `is_attr_data_in_db` detects existing attributes.
+    """
+    result = is_attr_data_in_db(User, {"email": "testuser@example.com"}, session=session)
+    assert result ==  "email already exists"
+    
+    is_attr_data_in_db(User, {"email": "sampleuser@example.com"}, session=session) is None
```

Ruff (0.8.2)

Pointless comparison at end of function scope. Did you mean to return the expression result?

(B015)